subprojects/core/src/main/java/org/gradle/api/internal/changedetection/state/OrderSensitiveTaskFilePropertyCompareStrategy.java

```diff
@@ -48,30 +47,32 @@ protected TaskStateChange computeNext() {
                         Map.Entry<String, NormalizedFileSnapshot> current = currentEntries.next();
                         String absolutePath = current.getKey();
```

Comment:
This would be better as `currentAbsolutePath` since all the other local variables now conform to that naming convention.